themes/default/content/blog/nodejs-binaries-with-pkg/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Node.js Native Binary Compilation Using vercel/pkg"
+date: 2022-09-13
+meta_desc: "Using Vercel pkg to create native binaries for nodejs providers"
```

Comment:
Oh, here's another difference. We should pick one and use the same terminology consistently.